pip/locations.py

```diff
@@ -127,6 +126,9 @@ def virtualenv_no_global():
     for path in appdirs.site_config_dirs('pip')
 ]
 
+venv_config_file = os.path.join(sys.prefix, config_basename)
+new_config_file = os.path.join(appdirs.user_config_dir("pip"), config_basename)
```

Comment:
Moved it from https://github.com/pypa/pip/pull/4240/files/8a680eb11875e8d4717881cc063b522fd67736d8#diff-576193e4169f8d7331131eb59ff4c840L103